pkg/rancher/rancher.go

```diff
@@ -464,7 +464,7 @@ func setupRancherService(ctx context.Context, restConfig *rest.Config, httpsList
 // This should only be called when Rancher is run in a Docker container because the Kubernetes version and Rancher version
 // are bumped at the same time. In a Kubernetes cluster, usually the Rancher version is bumped when the cluster is upgraded.
 func bumpRancherWebhookIfNecessary(ctx context.Context, restConfig *rest.Config) error {
-	webhookVersionParts := strings.Split(os.Getenv("CATTLE_RANCHER_WEBHOOK_MIN_VERSION"), "+up")
+	webhookVersionParts := strings.Split(os.Getenv("CATTLE_RANCHER_WEBHOOK_VERSION"), "+up")
```

Comment:
We don't need to update this value because this case shouldn't be hit on versions of Rancher > 2.7.3 because we do not support K8s 1.22+.